node/src/actors/chain_manager/handlers.rs

```diff
@@ -886,6 +886,11 @@ impl Handler<PeersBeacons> for ChainManager {
             peers_beacons.ignore_no_consensus_beacons(last_superblock_consensus);
         }
 
+        if self.current_epoch.is_none() {
+            return Err(());
+        }
+        let current_epoch = self.current_epoch.unwrap();
```

Comment:
How is this possible? Because of an uninitialized the `EpochManager`?

Reply:
yes